compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -84,6 +83,37 @@ impl<'a> Interpreter<'a> {
         Ok(result)
     }
 
+    fn call_builtin(
+        &mut self,
+        function: FuncId,
+        arguments: Vec<(Value, Location)>,
+        location: Location,
+    ) -> IResult<Value> {
+        let attributes = self.interner.function_attributes(&function);
+        let func_attrs = attributes.function.as_ref()
+            .expect("all low level functions must contain a function  attribute which contains the opcode which it links to");
```

Comment:
nit: should this be `all builtin functions must ..` because `low level` functions aren't distinguished from other builtins until later?

Reply:
Fixed